build.gradle

```diff
@@ -19,6 +19,18 @@ plugins {
   id 'idea' // Support for IntelliJ IDEA projects
 }
 apply plugin: 'kotlin'
+apply plugin: 'jacoco'
+jacoco {
+    toolVersion = "0.7.6.201602180812"
+    reportsDir = file("build/customJacocoReportDir")
+}
+jacocoTestReport {
+    reports {
+        xml.enabled false
+        csv.enabled false
+        html.destination "build/jacocoHtml"
```

Comment:
What is the default value for the report destination? 
And what about using this: `html.destination "${buildDir}/jacocoHtml"`?